tools/python/requirements.txt

```diff
@@ -1,16 +1,4 @@
-attrs==17.4.0
-boutiques==0.5.8
-certifi==2018.4.16
-chardet==3.0.4
-funcsigs==1.0.2
-functools32==3.2.3.post2
-idna==2.6
 jsonschema==2.6.0
-more-itertools==4.1.0
-pluggy==0.6.0
-py==1.5.3
 pytest==3.5.0
 requests==2.18.4
-simplejson==3.13.2
-six==1.11.0
-urllib3==1.22
```

Comment:
I'd like to have @gkiar's opinion here. I thought there was some rationale in expliciting the dependencies through frequent `pip freeze > requirements.txt`.

Reply:
Approved during the meeting!